css-brochure-form/index.html

```diff
@@ -0,0 +1,52 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <link rel="preconnect" href="https://fonts.googleapis.com">
+  <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+  <link href="https://fonts.googleapis.com/css2?family=Baloo+Thambi+2&display=swap" rel="stylesheet">
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <meta http-equiv="X-UA-Compatible" content="ie=edge">
+  <title>CSS Challenge - Brochure Form</title>
+  <link rel="stylesheet" href="fonts.css">
+  <link rel="stylesheet" href="reset.css">
+  <link rel="stylesheet" href="styles.css">
+</head>
+<body>
+  <div class="container">
+
+    <h3>LEARN MORE</h3>
+    <div>
+      <div>
+        <h6>COMPANY EMAIL</h6>
+        <input type="text" name="email" placeholder="sally@company.co">
+      </div><br />
+      <div>
+        <h6>NAME</h6>
+        <input type="text" name="name" placeholder="Sally Smith">
+      </div><br />
+      <div>
+        <h6>WHERE ARE YOU HIRING?</h6>
+        <input type="text" name="hiring" placeholder="City,Country">
+      </div>
+      <br />
+    </div>
+    <div>
+      <h6>HOW MANY ENGINEERS ARE YOU HIRING IN THE NEXT 6 MONTHS?</h6>
+      <form>
+        <select class="dropdown">
+          <option value="1-2Engineers">1-2 Engineers</option>
+          <option value="1Engineer">1</option>
+          <option value="2Engineer">2</option>
+        </select>
+      </form>
+      <br/>
```

Comment:
Please remove all `<br/>` elements, they should not be used to create space between elements, they should only be used to break text to a new line after a specific word within a text element.